pallets/asset-handler/src/lib.rs

```diff
@@ -428,6 +433,22 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Withdraw Fee
```

Comment:
Please extend documentation with params description as we have in other extrinsics